documentation/testing/security-testing.md

```diff
@@ -0,0 +1,19 @@
+# Security and Penetration Testing
+
+This document is designed to outline the current approach to security and penetration testing.
+
+## External IT Health Check / Penetration Testing
+
+An external 3rd party annually carry out a multiphase assessment of the Web Applications, APIs and Azure environment utilised by the Department for Education.
```

Comment:
are the annual IT health checks will be done once we go live? can you add a bit more context here please.

Reply:
This is as far as I'm aware - it's not a one and done.